src/assertions/OrderAssertions.ts

```diff
@@ -146,29 +174,34 @@ export class OrderAssertions {
         );
       }),
     );
-
-    if (relayerToken !== NULL_ADDRESS) {
-      await this.erc20Assertions.implementsERC20(relayerToken);
-    }
   }
 
   public async assertLiquidityOrders(
```

Comment:
Let's make this `assertLiquidityValidity`, time to start omitting the `order` terminology